### DISCUSSION TITLE
`cue import` and glob

### DISCUSSION THREAD

**Discussion by magick93** [cue-lang/cue#2315] [General]
Hello

I believe `cue import` can import multiple files. 

I'm attempting to import all `*.json` files in a directory using `cue import -f -l '#Communication:' /home/anton/git/my/apps/hr/generators/communication/json/*.json  -p communication -o communication.cue`

However this results in only one file being imported. 

Is this a bug, or am I misunderstanding the capabilities?

**Comment by myitcv** [comment]
@magick93 can you share what you have tried?

Here is an example of this working:

```
exec bash -c 'cue import -f -l "values:" -l "key" -p test *.json'
cmp x.cue x.cue.golden
cmp y.cue y.cue.golden

exec cue export .
cmp stdout stdout.golden

-- x.json --
{
	"key": "x",
	"name": "An X thing"
}
-- y.json --
{
	"key": "y",
	"name": "An Y thing"
}
-- x.cue.golden --
package test

values: x: {
	key:  "x"
	name: "An X thing"
}
-- y.cue.golden --
package test

values: y: {
	key:  "y"
	name: "An Y thing"
}
-- stdout.golden --
{
    "values": {
        "x": {
            "key": "x",
            "name": "An X thing"
        },
        "y": {
            "key": "y",
            "name": "An Y thing"
        }
    }
}
```

Points to note from the example:

* The shell-like language here is not exactly bash, but `testscript`. For more details on that please see https://github.com/cue-lang/cue/wiki/Creating-test-or-performance-reproducers, however that's not critical to understanding the example.
* If you have multiple JSON files that would conflict if imported, then you have to tell CUE where to place them. This you can do with the `-l` flag as shown.

If you have a concrete example of what you tried then maybe we can help to debug more.
